src/components/QuickGrid/QuickGrid.Props.ts

```diff
@@ -6,23 +6,31 @@ export interface IQuickGridProps {
     groupBy: Array<string>;
     gridClassName?: string;
     headerClassName?: string;
-    sortColumn?: string;
-    sortDirection?: 'ASC' | 'DESC';
     rowHeight: number | ((info: { index: number }) => number); // Number or a function that returns the height of a row given its index
     headerHeight: number;
     overscanRowCount?: number;
-    highlightHoverRow?: boolean;
     onSelectedRowChanged?: (selectedRowIndex: number) => void;
     onRowDoubleClicked?: (row: any) => void;
+    displayGroupContainer?: boolean;
+
+    sortColumn?: string;
+    sortDirection?: 'ASC' | 'DESC';
+    onGroupByChanged?: (groupBy: Array<string>) => void;
+
+    groupBySortColumn?: string;
+    groupBySortDirection?: 'ASC' | 'DESC';
```

Comment:
if the underlying library (lodash?) wants 'asc' and 'desc' should we also change to that? But it seems to me that usage of enum would be much cleaner.